lib/tls.js

```diff
@@ -3,6 +3,7 @@
 const net = require('net');
 const url = require('url');
 const util = require('util');
+const binding = process.binding('crypto');
```

Comment:
I stole it from [the](https://github.com/iojs/io.js/blob/804e7aa9ab0b34fa88709ef0980b960abca5e059/lib/os.js#L3) [other](https://github.com/iojs/io.js/blob/372bf83818fd87bae5ad7cedadfa121ddc5d4345/lib/zlib.js#L4) modules - I originally had it as `cryptoBinding`.